tests/integration/exec.bats

```diff
@@ -57,3 +57,42 @@ function teardown() {
   [[ ${lines[1]} == *"."* ]]
   [[ ${lines[2]} == *".."* ]]
 }
+
+@test "runc exec ls -la with --cwd" {
+  # run busybox detached
+  runc run -d --console /dev/pts/ptmx test_busybox
+  [ "$status" -eq 0 ]
+
+  wait_for_container 15 1 test_busybox
+
+  runc exec --cwd /bin test_busybox pwd
+  [ "$status" -eq 0 ]
+
+  [[ ${output} == *"/bin"* ]]
```

Comment:
Well, looks like it comes from `bats` itself, and they denial to say it is an issue, https://github.com/sstephenson/bats/issues/78 , so we would live with that.